vermouth/pdb/pdb.py

```diff
@@ -265,6 +265,38 @@ def _atom(self, line, lineno=0):
     atom = _atom
     hetatm = _atom
 
+    def cryst1(self, line, lineno=0):
+        """
+        Parse the CRYST1 record. Crystal structure information are stored with
+        the parser object and may be extracted later.
+        """
+        fields = [
+            ('', str, 6),
+            ('a', float, 9),
+            ('b', float, 9),
+            ('c', float, 9),
+            ('alpha', float, 7),
+            ('beta', float, 7),
+            ('gamma', float, 7),
+            ('space_group', str, 11),
+            ('z_value', int, 4),
+            ]
+        start = 0
+        field_slices = []
+        for name, type_, width in fields:
+            if name == "space_group":
+                start+=1
```

Comment:
lint.
Also, why this +1? If there's space in the format, take a look at how I solved it with the ATOM records. IIRC I put a negative width

Reply:
yeah there is a gap in the fields between space_group and the next item.